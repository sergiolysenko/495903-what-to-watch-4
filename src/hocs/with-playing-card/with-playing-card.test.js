import React from "react";
import renderer from "react-test-renderer";
import withPlayingCard from "./with-playing-card.js";

const mockComponent = () => <div />;
const mockFunck = () => {};

const MockComponentWrapped = withPlayingCard(mockComponent);

it(`render withPlayingCard`, () => {
  const tree = renderer.create(
      <MockComponentWrapped
        isPlaying={false}
        onMouseEnter={mockFunck}
        onMouseLeave={mockFunck}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
