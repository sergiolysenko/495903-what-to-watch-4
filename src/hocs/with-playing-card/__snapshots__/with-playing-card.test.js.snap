// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render withPlayingCard 1`] = `<div />`;
